implementation/src/main/module/package-info.java

```diff
@@ -0,0 +1,27 @@
+module io.smallrye.mutiny {
```

Comment:
I'd really call it `module-info.java`, because `package-info.java` is a completely different thing :-)

Reply:
oh right! Good point!

Reply:
Fixed!

Reply:
You might want to change the commit message, but I realize I'm just being picky here :grin:

Reply:
no, it must be changed.
I've done it already.

Reply:
Looks fine now, I probably hit some GitHub caching. Sorry about the noise :-)